### ISSUE TITLE
Profile picture not showing when someone creates or edits their profile on external website

### ISSUE THREAD

**Issue by hillsideassembly** [SparkDevNetwork/Rock#5579]
### Please go through all the tasks below

- [X] Check this box only after you have successfully completed both the above tasks

### Please provide a brief description of the problem. Please do not forget to attach the relevant screenshots from your side.

We are using the Family Pre Registration block for people to create or edit their family's information from the external website. We enabled the block's setting for uploading a profile picture. However, after they do so, it does not show on their profile page in Rock's internal website. Instead we get an error message that says the source is null.

### Expected Behavior

People should be able to upload a profile picture and it should show under the profile.

### Actual Behavior

People can upload a picture, but it does not show on their profile, instead a broken image is displayed.

### Steps to Reproduce

Add "Family Pre Registration" block to a page on external website
Edit block settings: Allow Updates: Yes. Auto Match: Yes. Profile Photos: Show.
Log in and edit profile or create a new profile using that block. Add a profile photo. Save.
Note: if you are using an admin profile, rock may create a duplicate profile, which will have the broken image.
Did this on the demo site:
https://rock.rocksolidchurchdemo.com/page/211
https://rock.rocksolidchurchdemo.com/Person/69

### Rock Version

1.15.1.1

### Client Culture Setting

en-US

**Comment by hillsideassembly** [comment]
This is the error that comes back:
ArgumentNullException in SixLabors.ImageSharp
Message
Parameter "source" must be not null. Parameter name: source

Stack Trace
at SixLabors.ThrowHelper.ThrowArgumentNullException(String name, String message)
at SixLabors.ThrowHelper.ThrowArgumentNullExceptionForNotNull(String name)
at SixLabors.ImageSharp.Processing.ProcessingExtensions.Mutate(Image source, Configuration configuration, Action`1 operation)
at Rock.ExtensionMethods.CropResize(Image source, Int32 width, Int32 height) in \Rock\Utility\ExtensionMethods\Image.cs:line 106
at Rock.Drawing.Avatar.AvatarHelper.CreateAvatar(AvatarSettings settings) in \Rock\Drawing\Avatar\AvatarHelper.cs:line 149
at RockWeb.GetAvatar.ProcessRequest(HttpContext context)
at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
at System.Web.HttpApplication.ExecuteStepImpl(IExecutionStep step)
at System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)

**Comment by hillsideassembly** [comment]
If you go to edit the profile, the picture shows there, but it doesn't show on their main profile page.

**Comment by karenn-ccbc** [comment]
I just added a related issue in regards to adding a photo via a workflow https://github.com/SparkDevNetwork/Rock/issues/5698 that has the same results no profile image displaying.

**Comment by chead4** [comment]
@karenn-ccbc  Thanks for creating the new issue Karen.  We will add comments on https://github.com/SparkDevNetwork/Rock/issues/5698 going forward.
